src/asm/rgbasm.1

```diff
@@ -7,89 +7,187 @@
 .\"
 .Dd July 8, 2019
 .Dt RGBASM 1
-.Os RGBDS Manual
+.Os
 .Sh NAME
 .Nm rgbasm
 .Nd Game Boy assembler
 .Sh SYNOPSIS
-.Nm rgbasm
-.Op Fl EhLVvw
+.Nm
 .Op Fl b Ar chars
 .Op Fl D Ar name Ns Op = Ns Ar value
+.Op Fl E
 .Op Fl g Ar chars
+.Op Fl h
 .Op Fl i Ar path
+.Op Fl L
```

Comment:
Splitting up flags that don’t take arguments goes against normal practice in a manpage synopsis.

> [Options without operands come first, in alphabetical order inside a single set of braces, followed by options with operands, in alphabetical order, each in braces, followed by required arguments in the order they are specified, followed by optional arguments in the order they are specified.](https://man.openbsd.org/style.9)

The alphabetically sorted description of all flags is present instead in the `DESCRIPTION` section.

Reply:
I looked at other man pages, and some (`less`) did follow that, while others (`git` subcommands) didn't. So I picked the one that looked the most user-friendly. If it's standard, though, I will revert. I'd be curious to know the rationale behind the Git man pages, though.

Reply:
I wrote from the BSD perspective, but there are variations, such as POSIX’s [Utility Argument Syntax](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_01) (non‐normative, I think) that suggests individual alphabetical options for programs with few arguments and grouped options for programs with many. In my experience more software groups options than splits them. (A lot of stuff just leaves out the options completely like `program_name [options] file ...`which is just the *worst*.)

I imagine that both `git` and `less` came up with their documentation practices on the fly and weren’t following POSIX or BSD guidelines.